### DISCUSSION TITLE
Disable Jekyll through settings

### DISCUSSION THREAD

**Discussion by axelboc** [community/community#57010] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've spent a whole morning trying to fix a broken Storybook v7 site caused by GitHub Pages skipping a JS file starting with an underscore... To fix it, I had to change my CI workflow to [create a `.nojekyll` file](https://github.com/storybookjs/storybook/issues/20564) after building the Storybook site.

People keep [running](https://github.com/orgs/community/discussions/55035) [into](https://github.com/orgs/community/discussions/32137) [this](https://github.com/orgs/community/discussions/33458) [issue](https://github.com/storybookjs/storybook/issues/19065)... 

Please, please, please allow us to disable Jekyll through the GitHub Pages settings!

**Comment by yoannchaudet** [comment]
We actually do. Not the way you expect though.

This is how we want more people to configure their site: https://github.blog/changelog/2022-07-27-github-pages-custom-github-actions-workflows-beta.

The branch model is our "old" way of doing things where we would assume you are using Jekyll.

**Reply by axelboc** [reply]
Riiight, that makes sense. Thanks for the tip!
